core/domain/takeout_service.py

```diff
@@ -18,15 +18,18 @@
 from __future__ import absolute_import   # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import inspect
 import re
 
 from core.platform import models
 
 (
-    collection_models, email_models, exploration_models, feedback_models,
-    suggestion_models, user_models) = models.Registry.import_models([
-        models.NAMES.collection, models.NAMES.email, models.NAMES.exploration,
-        models.NAMES.feedback, models.NAMES.suggestion, models.NAMES.user])
+    base_models, collection_models, email_models, exploration_models,
```

Comment:
Try to make the line breaks here match those in the list passed to import_models(), so that it is easy to compare the two.

Reply:
@varun-tandon, I think this hasn't been resolved yet.

Reply:
Resolved in latest commit. Let me know if I misunderstood!